### ISSUE TITLE
Establish & Configure Testing Environment

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#17324]
- [x] Import Sample Org Data
- [x] Copy Environment
- [x] Configure User Testing Feedback Form
- [x] Review Code in Testing Environment
- [x] Review Emails in Testing Environment
- [x] Show/Hide appropriate Views

**Comment by ChrispinP** [comment]
Environment successfully created. Reviewed code and updated urls.

Also prepared the User Testing Feedback Form
![image](https://github.com/cityofaustin/atd-data-tech/assets/67915167/4ae6f821-6248-4d9a-8ca7-e7db8cd50183)

**Comment by KaroEngstrom** [comment]
### Notes

* Details - deleted old views
     * If Status is Review Completed and the Review Type Flag is Single Review, hide menu (these the pre-submission cycle records)
* Submission Details - deleted old views 
* TCP landing page, no changes on the grids
* CSWZ! Must update old grid to the new one and delete new, otherwise the child pages get moved up to TCP

**Comment by ChrispinP** [comment]
Testing Environment views have been updated. Moved the ROW Portal dropdown menu to the top of the page structure so customers who click on the banner will not be directed to the Staff Home. Customer Dashboard menu updated with child pages appropriately.

Additionally, cleaned up some language, urls, view & page names.

**Comment by ChrispinP** [comment]
All emails were updated, a couple were missing so I added them as well. Links in all emails for the Test environment will direct to itself and currently are set to go to Karo & Chrispin until we are ready to test real email field values.

Also connected CSWZ requests to the Org object so those email fields are visible both in Test & Prod environments.

Based on associated tasks, the Test environment is now configured and is ready for user testing.
